### ISSUE TITLE
licensing clarification

### ISSUE THREAD

**Issue by zabbal** [labnation/DeviceInterface#2]
Please add LICENSE or COPYING file to clarify under which license the source code in this repo is available.

**Comment by rubdos** [comment]
Yes please! I'd like to look at the code, but I can't as long as it's not been signified what license this is under.

**Comment by JoelStienlet** [comment]
Sorry to reopen this topic five years later, but the licensing is still unclear. Can you please clarify this matter?

**Comment by zabbal** [comment]
Nowadays the preferred way to specify license is SPDX.

**Comment by ttlns** [comment]
@labnation Are there any updates regarding this planned?
